src/info/langs/language.tera

```diff
@@ -109,6 +110,20 @@ impl Language {
     }
 }
 
+/// Compiles the top-level lines-of-code of a tokei `Language`. Takes into
+/// account that a prose language's comments *are* its code.
+pub fn compile_tokei_loc(language_type: &tokei::LanguageType, language: &tokei::Language) -> usize {
+    match language_type {
+        {% for language, attrs in languages -%}
+            tokei::LanguageType::{{ language }} => {
+                {% set line_types = attrs.tokei_line_types | default(value=['code']) %}
+                language.{{ line_types.0 }}{% for line_type in line_types | slice(start=1) %} + language.{{ line_type }}{% endfor %}
+            },
```

Comment:
```suggestion
            {% set line_types = attrs.tokei_line_types | default(value=['code']) -%}
            tokei::LanguageType::{{ language }} => language.{{ line_types.0 }}{% for line_type in line_types | slice(start=1) %} + language.{{ line_type }}{% endfor %},
```

  minor: use an inline match syntax to reduce clutter:
  
  ```rust
      match language_type {
        tokei::LanguageType::ABNF => language.code,
        tokei::LanguageType::Abap => language.code,
        tokei::LanguageType::Ada => language.code,
        tokei::LanguageType::Assembly => language.code,
        tokei::LanguageType::AutoHotKey => language.code,
....
  ```

Reply:
Good point, I'm a bit out of practice with Rust and forgot that the block was optional 😆